app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -70,16 +70,14 @@
               <dt>My Rewards</dt>
               <dd>n/a</dd>
             </dl>
-            <div class="validator-profile__header__data__break" />
+            <div class="page-profile__header__data__break" />
             <dl class="colored_dl">
               <dt>Voting Power</dt>
-              <dd id="validator-profile__power">
-                {{ pretty(powerRatio * 100) }} %
-              </dd>
+              <dd id="page-profile__power">{{ pretty(powerRatio * 100) }} %</dd>
             </dl>
             <dl v-if="config.devMode" class="colored_dl">
               <dt>Uptime</dt>
-              <dd id="validator-profile__uptime">
+              <dd id="page-profile__uptime">
```

Comment:
are you sure about name spacing these under `page-profile`?

Reply:
i think it's an ok name, intended to describe our pages with a "profile-like" layout. open to suggestions if you have them.

Reply:
If we recycle all this I'm happy! 
The `uptime` case seems a bit contextualized in validators although, everything else seems general enough to be recycled in other pages.

Ideally I would keep separate (declare in validator page) only those three classes:
- self-bond 
- commissions
- uptime

Reply:
yeah, i agree with you @sabau - i didn't actually write or refactor any css in this PR - just renamed things. i would be happy to improve the css here to make it more generalized or i could do this in another PR ... whatever you guys think is best.

Reply:
I am fine with doing it in another PR. Going to merge this. Can you open a follow up?